src/js/utils/updateTabList.js

```diff
@@ -12,7 +12,7 @@ export function updateTabList(features) {
         isExpertModeEnabled && FC.CONFIG?.buildOptions?.includes("USE_SERVOS"),
     );
 
-    $("#tabs ul.mode-connected li.tab_gps").toggle(features.isEnabled("GPS"));
+    $("#tabs ul.mode-connected li.tab_gps").toggle(isExpertModeEnabled && FC.CONFIG?.buildOptions?.includes("USE_GPS"));
```

Comment:
GPS is not showing on virtualFC in this PR. It does show in master.

Reply:
is this indeed for the configation tab toggle? if so, would rather not require expert-mode -- will confuse pilots.

Reply:
- Does show here (the first time):
![image](https://github.com/user-attachments/assets/cb80bf93-c429-4a24-b5ed-9a24a8d75b1b)
- Now switching expert mode on off on it removes even more tabs.

Reply:
this: 
![image](https://github.com/user-attachments/assets/c6b7888a-530f-4644-a7dd-1b3a6a0264b4)

Reply:
i dont know why but `ad449a6d` is still as above sreenshot.

Reply:
Did you mean to remove DISPLAY ?

Reply:
non i mean `GPS` does not exist here

Reply:
Yes correct - the GPS feature has been moved to GPS tab now to be able to configure MSP GPS without having to activate GPS sensor in ports tab (stills needs to enable MSP on the UART used off course).

Reply:
oh, okay. i did not follow that idea.  interesting. 
What if it remains in both locations, since "features" were always in configuration tab?
This should be discussed by all devs, i feel.

Reply:
Already did the same for receiver, motor and pid tuning tab